### ISSUE TITLE
No way to force color output (when piping to less)?

### ISSUE THREAD

**Issue by hartwork** [koalaman/shellcheck#294]
Hi!
I'm trying to pipe shellcheck output to less while maintaining color on the display.
No luck with `shellcheck -f tty filename | less -r`: all black and white.
Is is possible?
Thanks!

**Comment by koalaman** [comment]
There's no built-in way to force colors to non-ttys, but you can trick shellcheck with `script`:

    script -q -c 'shellcheck filename' < /dev/null | less -r

**Comment by hartwork** [comment]
```
# script -q -c 'shellcheck /usr/sbin/debootstrap' < /dev/null | less -r
script: cannot open typescript: Permission denied
Terminated
```
Any ideas?

Please take my vote for a command line parameter `--color=force` or so.

**Comment by koalaman** [comment]
Fixed in @haguenau's a3d4101

**Comment by hartwork** [comment]
Thank you!
